src/process.rs

```diff
@@ -446,6 +447,99 @@ pub struct Io {
     pub cancelled_write_bytes: u64,
 }
 
+/// Mount information from `/proc/<pid>/mountstats`.
+#[derive(Debug, PartialEq)]
+pub struct MountStat {
+    device: Option<String>,
+    mount_point: PathBuf,
+    fs: String,
+    statistics: Option<MountNFSStatistics>,
+}
+
+impl MountStat {
+    pub fn from_reader<R: io::Read>(r: R) -> Vec<MountStat> {
+        vec![]
+    }
+}
+
+/// Only NFS mounts provide additional statistics in `MountStat` entries.
+#[derive(Debug, PartialEq)]
+pub struct MountNFSStatistics {
+    version: String,
+    // * opts: HashMap<String, Some(String)>
+    /// Duration the NFS mount has been in existence.
+    age: Duration,
+    // * fsc (?)
+    // * impl_id (NFSv4): Option<HashMap<String, Some(String)>>
+    // * caps: HashMap<String, String>
+    // * nfsv4 (NFSv4): Option<HashMap<String, Some(String)>>
+    // * sec: HashMap<String, Some(String)>
+    events: NFSEventCounter,
+    bytes: NFSByteCounter,
+    // * RPC iostats version:
+    // * xprt
+    // * per-op statistics
+    per_op_stats: NFSPerOpStats,
+}
+
+#[derive(Debug, PartialEq)]
```

Comment:
Why `PartialEq`?

Reply:
Basically two reasons:

 * `PartialEq`is required for the tests to compare the result
 * `Eq` can be added since I see no reason why this is not reflexive. But `Eq` doesn't add any functions, so `PartialEq` is still required.

Reply:
Ah, good point about needed to compare them in tests.   Does it make sense to derive `PartialEq` only when we're running tests?   My only concern (which is pretty minor) is that a public PartialEq derivation suggests that it makes sense for a user to compare these objects, which I don't think is true.

While on the same subject, I wonder if these structs should get `Copy` and `Clone`

Reply:
That is a good point, I'll add it to the Todo-list